src/main/java/ca/team2706/frc/robot/Robot.java

```diff
@@ -159,6 +159,8 @@ public void testPeriodic() {
      * @param args Arguments passed on startup
      */
     public static void main(String[] args) {
+        Logging.init();
```

Comment:
Should this be called in `robotInit()` or is there a specific reason why it's called here instead?